docs/starters.yml

```diff
@@ -1,3 +1,69 @@
+- url: https://gatsby-starter-blog-demo.netlify.com/
```

Comment:
What are these changes?

Reply:
Looks like he has moved 5 ones that always work to the top

Reply:
Ha -- I'm so sorry I missed these!

I did this for a reason :) I'd like to eventually make it easier to get up and running with .org, and popping off the top 5 seems like an easy approach 🎉